codegen/gen-crd-client.sh

```diff
@@ -71,5 +71,5 @@ function generate_client() {
   rm -rf ${TEMP_DIR}
 }
 
-echo "##### Generating config.openservicemesh.io client ######"
-generate_client "config" "v1alpha1"
\ No newline at end of file
+echo "##### Generating osmconfig.openservicemesh.io client ######"
+generate_client "osmconfig" "v1alpha1"
```

Comment:
This should be `config` because that's the group prefix for the group `config.openrservicemesh.io`